airflow/include/tasks/split.py

```diff
@@ -103,3 +103,11 @@ def split_html(dfs: list[pd.DataFrame]) -> pd.DataFrame:
     df.reset_index(inplace=True, drop=True)
 
     return df
+
+
+def split_list(urls: list, chunk_size: 0):
```

Comment:
```suggestion
def split_list(urls: list[str], chunk_size: int = 0) -> list[str]:
```

Reply:
return would be list[list] just fixed it PTAL